tests/test_isrTaskLSST.py

```diff
@@ -257,19 +270,19 @@ def test_isrBootstrapFlat(self):
         self.assertFloatsAlmostEqual(ratio[good_pixels], self.flat_adu.image.array[good_pixels], atol=1e-5)
 
         # Test the variance plane in the case of adu units.
-        # The expected variance stars with the image array.
+        # The expected variance starts with the image array.
         expected_variance = result.exposure.image.clone()
         # We have to remove the flat-fielding from the image pixels.
         expected_variance.array *= self.flat_adu.image.array
-        # And add the gain and read noise (in adu) per amp.
+        # And add the gain and read noise (in electron) per amp.
         for amp in self.detector:
             # We need to use the gain and read noise from the header
             # because these are bootstraps.
             gain = result.exposure.metadata[f"LSST ISR GAIN {amp.getName()}"]
             read_noise = result.exposure.metadata[f"LSST ISR READNOISE {amp.getName()}"]
 
             expected_variance[amp.getBBox()].array /= gain
-            expected_variance[amp.getBBox()].array += read_noise**2.
+            expected_variance[amp.getBBox()].array += (read_noise/gain)**2.
```

Comment:
I think a comment should be added here that the conversion here is necessary because this is a bootstrap version of the variance plane just to be extra sure.